include/cucumber-cpp/internal/step/BreakpointSupport.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef CUKE_BREAKPOINTSUPPORT_HPP_
+#define CUKE_BREAKPOINTSUPPORT_HPP_
+
+#include "../CukeDll.hpp"
+
+namespace cucumber {
+namespace internal {
+
+#if defined(_MSC_VER)
+// The ", true" makes the expression to be of type bool, which is required
+// for the CUKE_BREAK_INTO_DEBUGGER macro
+#define CUKE_BREAK_INTO_DEBUGGER (__debugbreak(), true)
+
+#elif defined(__clang__) && defined(__APPLE__) && !TARGET_OS_IPHONE
```

Comment:
I'm not 100% sure. Won't the clang pragmas cause issues with other compilers? IIRC gcc's `-Wall` also includes `-Wunknown-pragma` (I might be mistaken here).